bower.json

```diff
@@ -23,24 +23,24 @@
     "test"
   ],
   "dependencies": {
-    "purescript-aff": "^5.0.0",
-    "purescript-either": "^4.0.0",
-    "purescript-free": "^5.0.0",
-    "purescript-maybe": "^4.0.0",
+    "purescript-aff": "^5.1.1",
+    "purescript-either": "^4.1.1",
+    "purescript-free": "^5.2.0",
+    "purescript-maybe": "^4.0.1",
     "purescript-newtype": "^3.0.0",
-    "purescript-prelude": "^4.0.0",
+    "purescript-prelude": "^4.1.1",
     "purescript-tailrec": "^4.0.0",
-    "purescript-tuples": "^5.0.0",
+    "purescript-tuples": "^5.1.0",
     "purescript-type-equality": "^3.0.0",
     "purescript-unsafe-coerce": "^4.0.0",
-    "purescript-variant": "^6.0.0",
+    "purescript-variant": "^6.0.1",
     "purescript-profunctor": "^4.0.0",
-    "purescript-effect": "^2.0.0"
+    "purescript-effect": "^2.0.1"
   },
   "devDependencies": {
-    "purescript-control": "^4.0.0",
+    "purescript-control": "^4.1.0",
     "purescript-minibench": "^2.0.0",
-    "purescript-identity": "^4.0.0",
-    "purescript-console": "^4.1.0"
+    "purescript-identity": "^4.1.0",
+    "purescript-console": "^4.2.0"
```

Comment:
Is this just a general dependency bump? I think that it was only relying on transitive `typelevel-prelude` imports, and the dependency ranges should otherwise be valid. With the transitive import, I don't think there's any problem compiling it on 0.13. In which case I think only the import changes are necessary, and this can be a patch bump.